packages/cli/test/unit/commands/project.test.ts

```diff
@@ -28,29 +31,12 @@ describe('project', () => {
       data.pop();
 
       expect(org).toEqual(user.username);
-      expect(header).toEqual(['name', 'updated']);
-      expect(data).toEqual([project.project.name]);
-    });
-    it('should list deployments for a team', async () => {
-      useUser();
-      const team = useTeams('team_dummy');
-      const project = useProject({
-        ...defaultProject,
-      });
-
-      client.config.currentTeam = team[0].id;
-      client.setArgv('project', 'ls');
-      await projects(client);
-
-      const output = await readOutputStream(client, 2);
-      const { org } = pluckIdentifiersFromDeploymentList(output.split('\n')[0]);
-      const header: string[] = parseSpacedTableRow(output.split('\n')[2]);
-      const data: string[] = parseSpacedTableRow(output.split('\n')[3]);
-      data.pop();
-
-      expect(org).toEqual(team[0].slug);
-      expect(header).toEqual(['name', 'updated']);
-      expect(data).toEqual([project.project.name]);
+      expect(header).toEqual([
+        'Project Name',
+        'Latest Production URL',
+        'Updated',
+      ]);
+      expect(data).toEqual([project.project.name, `--`]);
```

Comment:
**suggestion (soft-blocking):** add test for deployment URL showing up in output

In this or a follow-up PR, can we test that a deployment URL makes it in as well?